src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch/README.md

```diff
@@ -0,0 +1,14 @@
+# OpenTelemetry.Instrumentation.Elasticsearch
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.Elasticsearch.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Elasticsearch)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.Elasticsearch.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Elasticsearch)
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Instrumentation.Elasticsearch
+```
+
+## References
+
+* [OpenTelemetry Project](https://opentelemetry.io/)
```

Comment:
I would add the elasticsearch url as well

Reply:
Done